### ISSUE TITLE
[Outreachy Task Submission] Accessibility issue with Collections modal form labels

### ISSUE THREAD

**Issue by Chiemezuo** [ushahidi/platform#4888]
# Intro
Accessibility-checking software flags the `Collection` modal when it opens after clicking on the `Collections` navigation button (by the side of every page) because there is no appropriate label for the search field.

## How to reproduce this issue.
If you use a tool like [WAVE evaluation tool](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh), you can follow these steps:

1. Open your browser and log into your running Ushahidi deployment (from localhost).
2. Click on `Collections` by the left side of the screen.
3. When the `Collection` modal opens, put on the Accessibility browser extension. It will flag the form input field.

## Accessibility Violation
Success Criterion 3.3.2 Labels or Instructions

 (Level A)
[Labels](https://www.w3.org/TR/WCAG22/#dfn-labels) or instructions are provided when content requires user input.
